Workflow.Rmd

```diff
@@ -368,85 +370,351 @@ state_accounts_index %>%
+are just as applicable to economics as they are to pharmacometrics or natural language processing or computer vision. It's 
+all essentially the same! Fundamentally, any model has the following: 
+
+- A collection of "knowns"---variables that we treat as being known for sure, or assume to take
+particular values. If we're building a predictive model of whether a borrower defaults on a loan, this might be the information
+we have about the customer at the time we will be called on to make predictions. If we're a government choosing a fiscal policy, 
+it might be a projected spending pattern (even though this is not "known", for a modeling exercise we might pretend it is known
+for the sake of some policy simulation). In some cases, especially in time series and computer vision, 
+the set of "knowns" might be empty. We often use $X$ for "knowns". 
+- Some "model unknowns", normally called $\theta$. These might be latent variables, coefficients, scales, weights, 
+etc. Throughout this book we'll use the term "model unknown" rather than the more common "parameter", which we'll reserve
+for a specific use. In principal, we can build any model and plonk values in place of these model unknowns, and perform
+some analysis (thought that analysis might be bunk). In fact, many applications of economics---particularly 
+computable general equilibrium modeling---involve "calibrating" models in precisely this way, using 
+"reasonable" assumed values for model unknowns. Poorly chosen values
+of these unknowns will typically make the model say useless things, and most of this book is about strategies to come up
+with good values for these unknowns. 
+- A "model structure", often denoted by $\mathcal{M}$. This is a mathematical structure that provides some mapping between 
+the coupling of our knowns $X$ and unknowns $\theta$, and some outcomes that we're interested in, $y$. In economics, this might be
+as simple as linear model, or as complex as a large auction model or dynamic macroeconomic model. Outside economics, this
+could be a neural network for natural language processing, a Gaussian process for modeling voter preferences, or a 
+set of differential equations to model the impact of a drug on a biological system. 
+- A collection of "outcomes", $y$. These are the variables whose values we're interested in, either because we're trying to 
+generate predictions for them in the future, or because we're trying the generate predictions for them under different 
+policies or treatments. In economics these might be prices and sales, or macroeconomic variables like employment and 
+inflation. Outside economics the "outcome" might be whether a picture is of a "cat" or a "horse"; or perhaps
+whether an ill person recovers or dies of their illness.
+
+The sorts of models discussed in this book are *generative* or *probabilistic* models. Such models are probabilistic in the sense
```

Comment:
I think the explanation of "generative" and "probabilistic" could be improved. In "probabilistic" you're giving the key part now but it's missing the link to what people typically think of as relating to probability. What does it mean for a model/process to generate data (given knowns)? A clean example here might clear it up.

Reply:
Also maybe adding to, "there are many plausible outcomes that could occur", some notion of 'why' the particular outcome we saw did occur (e.g. it was a random draw from a particular system :p)